packages/frontend/src/features/sqlRunner/hooks/useTables.tsx

```diff
@@ -49,6 +49,7 @@ export const useTables = ({ projectUuid, search }: GetTablesParams) => {
                 projectUuid,
             }),
         retry: false,
+        enabled: !!projectUuid,
```

Comment:
same here

Reply:
probably this is relevant

Reply:
Both these changes (this on and the one above) fix the errors in the screenshot in the description, where the request was being performed when there was no `projectUuid` saved in the store. This is not needed and is mostly an addition to fix those errors which I noticed while testing the migration out

Reply:
okay cool!